### PR TITLE
remove jquery-mini $ syntax

### PR DESCRIPTION
**Pull request by wwwtyro**
@ibgreen :tada:

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+export function uid() {
+  return _uid++;
+}
+
+function _type(e) {
+  var t = Object.prototype.toString.call(e);
+  return t.substr(8, t.length - 9).toLowerCase();
+};
+
+export function type(elem) {
```
Comment (ibgreen) [inline src/utils.js]: Can we get rid of this function and just use instanceof JS keyword instead?

File: src/utils.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        mix[key] = merge(mp, op);
+      } else{
+        mix[key] = detach(op);
+      }
+    }
+  }
+  return mix;
+};
+
+export function extend(to, from) {
```
Comment (ibgreen) [inline src/utils.js]: Can we use Object.assign or "object spread" instead and drop this function?

File: src/utils.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+export function merge() {
```
Comment (ibgreen) [inline src/utils.js]: Could we add som js doc headers to the utilities since they won't be mentioned in user docs

File: src/utils.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    to[p] = from[p];
+  }
+  return to;
+};
+
+export function splat(a) {
+  return Array.isArray(a) && a || [a];
+};
+
+export function empty() {}
```
Comment (ibgreen) [inline src/utils.js]: Maybe rename to "noop"?

**Comment by ibgreen** [comment]
Very nice. Happy to pull but will wait for your response to comments

**Comment by ibgreen** [comment]
@wwwtyro I wanted the latest changes so I merged, just open a new PR if you have more changes.

**Comment by wwwtyro** [comment]
Ah okay, sorry about the delay; I will have lots of time this weekend.

**Pull request merged**